packages/components/psammead-most-read/src/Item/index.test.jsx

```diff
@@ -1,60 +1,64 @@
 import React from 'react';
 import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
 import { latin, arabic } from '@bbc/gel-foundations/scripts';
-import { MostReadLink, MostReadRank } from '.';
-import { getItem } from '../testHelpers/itemsHelper';
+import { MostReadLink } from '.';
+import { getItem, getItemWrapperArray } from '../testHelpers';
 
 describe('MostReadLink', () => {
   shouldMatchSnapshot(
     'should render ltr correctly',
     <MostReadLink
-      item={getItem('news')}
+      dir="ltr"
```

Comment:
`dir` is already defaulted to 'ltr' so can be omitted when you render `MostReadLink` here and in some other tests in this file

Reply:
I've added the 'ltr' here an in other places of the tests to make it clear that we are testing LTR direction.

Reply:
but i'll update it now to remove ltr where it is being defaulted

Reply:
updated now